frontend/src/App.js

```diff
@@ -436,15 +465,20 @@ class App extends Component {
               </Button>
             </div>
         </header>
+        <div class="tab">
+          <Button class="tablinks" onClick={() => this.getAllItems()}>All</Button>
+          <Button class="tablinks" onClick={() => this.getAllItems('[BBQ]')}>Barbecue</Button>
+          <Button class="tablinks" onClick={() => this.getAllItems('[TECH]')}>Tech Cabinet</Button>
+        </div>
         <div className="container">
           <h1>Information</h1>
-          <p>Welcome to the Engineering Undergraduate Society's rental platform for the tech cabinet.</p>
+          <p>Welcome to the Engineering Undergraduate Society's rental platform.</p>
           <p>To request an item, simply click on its name and follow the form instructions.</p>
           <p>Please contact Chris in the EUS Office (<b>McConnell room 7</b>) to collect your item.</p>
           <p>You are expected to return rented items within <b>five days</b> excluding week-ends.</p>
           <h1>Available Items</h1>
           <RentalTable tokenValidity={this.state.authLevel} results={this.state.results} deleteItem={this.deleteItem} reserveItem={this.reserveItem}/>
-          <div className="form" style={{display: this.state.email && this.state.authToken ? "block" : "none" }}>
+          <div className="form" style={{display: this.state.email && this.state.authToken && this.state.authLevel > 1 ? "block" : "none" }}>
```

Comment:
This was a bug in the earlier version: These buttons should only show if the user is an administrator (`authLevel` of 0 is logged out, 1 is logged in as user and 2 is logged in as admin).

It doesn't matter if a non-administrator can see these buttons because there's a server-side check preventing them from actually performing an action with them. This simple javascript just hides them to improve the overall UX.